0.61.1 Cherry Picks (#9999)
numba/numba: docs/source/release/0.61.1-notes.rst

Version 0.61.1rc1 (24 March 2025)
=================================

This is a maintenance release that adds support for NumPy 2.2 and fixes few bugs.

Highlights
~~~~~~~~~~

Support for NumPy 2.2
---------------------

> Numba now supports NumPy 2.2. following are some notable changes:

Reviewer: ```suggestion
Numba now supports NumPy 2.2, with the following notable changes:
```

Reviewer: or something similar
